docs/security.rst

```diff
@@ -0,0 +1,107 @@
+  Cross-Site scripting (XSS),✅,✅,ODK's developers are familiar with XSS best practices and consider it in code review.
+  Database Security,✅,❓,ODK Cloud databases are encrypted at rest and isolated by customer. Direct database access is not possible.
+  "Data Governance, Compliance, Privacy",✅,❓,"ODK Cloud is available in US or EU data centers. Data centers are GDPR compliant and ISO27K and SOC 2 certified. See `Terms of Service <https://getodk.org/tos>`_, `Privacy Policy <https://getodk.org/privacy>`_, and `Data Processing Agreement <https://getodk.org/dpa>`_."
+  Denial of Service (DOS),✅,❓,ODK Cloud monitors for attacks and mitigations can be put in place quickly. Automated DDoS protection is available.
+  Encryption,✅,✅,The ODK server requires encryption in transit (HTTPS). Additional :doc:`end-to-end encryption <central-encryption>` is available for low-trust environments. ODK Cloud adds encryption at rest.
+  Insurance,✅,❓,ODK Cloud is covered by General Liability and Professional Liability policies. Each has $2M/occurrence coverage.
+  Maintenance & Updates,✅,❓,ODK Cloud infrastructure automatically updates outdated or vulnerable software.
+  Manual Testing,✅,✅,ODK's QA process includes `manual testing <https://forum.getodk.org/t/how-the-qa-team-ensures-odk-is-reliable/49960>`_ both of new features and the full system before releases. Users also contribute manual testing of `betas <https://forum.getodk.org/c/releases/pre-releases/19>`_.
+  Password Storage,✅,✅,The ODK server uses BCrypt for password-hashing with a cost factor of 12.
+  Penetration Testing,✅,❓,"ODK Cloud has independent :ref:`security audits and penetration tests <security-audits>` that include testing OWASP Top 10 (e.g., broken access control, cryptographic failures, injection attacks, insecure design, misconfiguration)."
+  Physical Security,✅,❓,"ODK Cloud data centers have 24-hour security, video surveillance, limited network access, etc."
+  Roles & Permissions,✅,✅,"Web Users (e.g., project managers) can only be created with an email address and password resets are only possible through that email. App Users (e.g., data collectors) authenticate with a QR code and are fully isolated from all management functions. See :doc:`Central Users <central-users>` for more."
+  Single Sign-On (SSO) & Multi-Factor Auth (MFA),✅,✅,The ODK server offers :ref:`SSO via the OIDC protocol <central-install-digital-ocean-sso>`. MFA can be enabled at the identity provider.
+  SQL Injection (SQLi),✅,✅,The ODK server uses Slonik which is designed to prevent vulnerable queries. ODK's developers are familiar with SQLi best practices and consider it in code review.
+  SSL Certificates (HTTPS),✅,✅,The ODK server requires HTTPS and uses Let's Encrypt certs with TLS 1.3 and an `A+ rating from SSL Labs <https://www.ssllabs.com/ssltest/analyze.html?d=production.getodk.cloud>`_.
+  Uptime Management,✅,❓,"ODK Cloud has had `100% uptime <https://status.getodk.org/>`_ over the last 16 months."
```

Comment:
Not evergreen. Maybe something like "has experienced less than .001% downtime across any given 12 month period"